internal/cluster/transition.go

```diff
@@ -38,7 +38,7 @@ var resetFields = append(append(resetProgressFields, resetLogsField...), "opensh
 type transitionHandler struct {
 	log                 logrus.FieldLogger
 	db                  *gorm.DB
-	stream              stream.EventStreamWriter
+	stream              stream.EventNotifier
```

Comment:
Is this not used? It's strange to change the type, but not have any other changes in the file.

Reply:
After checking, I can confirm it is passed through as-is throughout the cluster transitionHandler. As receiver type also changed, no other change is needed.